dfx.json

```diff
@@ -4,21 +4,37 @@
       "main": "src/connectd/main.mo"
     },
     "linkedup": {
-      "main": "src/linkedup/main.mo",
+      "main": "src/linkedup/main.mo"
+    },
+    "linkedup_assets": {
+      "dependencies": ["linkedup"],
       "frontend": {
-        "entrypoint": "src/linkedup/public/main.js",
-        "assets": ["src/linkedup/public/*.html", "src/linkedup/public/*.css"]
-      }
+        "entrypoint": "src/linkedup/public/main.js"
+      },
+      "type": "assets",
+      "source": ["canisters/linkedup_assets/assets", "src/linkedup/public"]
```

Comment:
I don't think `"canisters/linkedup_assets/assets"` should be in here, because that's actually the destination where these sources are copied to.

Reply:
@ericswanson-dfinity so it should just be `"src/linkedup/public"` in there?

Reply:
Yes